### PR TITLE
Grid Component added

### PR DESCRIPTION
**Pull request by chanchal-panpaliya**
Grid Component added

### CODE REVIEW

**Review by way2nizam — APPROVED** [review]
code looks good.

File: Grid/Grid.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    'menu main main main right right'
+    'menu footer footer footer footer footer';
+  grid-gap: 1px;
+  padding: 1rem 1rem 1rem 1rem;
+  width: 100vh;
+  max-width: 100%;
+  height: auto;
+}
+
+.item1 { grid-area: header;  background-color: var(--background-color-danger); height: 10vh;}
```
Comment (Sharmaryan) [inline Grid/Grid.css:17]: this code is not formatted properly

File: Grid/Grid.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  background-color: var(--background-color-danger); 
+  position: relative;
+  width: fit-content;
+}
+
+.grid-3-item2{
+  background-color: var(--background-color-success); 
+  width: fit-content;
+  position: relative;
+
```
Comment (Sharmaryan) [inline Grid/Grid.css:64]: remove whitespaces

**Review by Sharmaryan — APPROVED** [review]
everything is fine, suggested small changes

**Pull request merged**